src/apidocs/ApiDocsArea.tsx

```diff
@@ -0,0 +1,230 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+} from "@chakra-ui/accordion";
+import { Box, BoxProps, Text } from "@chakra-ui/layout";
+import { Spinner } from "@chakra-ui/spinner";
+import sortBy from "lodash.sortby";
+import React, { useEffect, useMemo, useState } from "react";
+import {
+  apiDocs,
+  ApiDocsResponse,
+  ApiDocsBaseClass,
+  ApiDocsEntry,
+} from "../language-server/apidocs";
+import { useLanguageServerClient } from "../language-server/language-server-hooks";
+import { pullModulesToTop } from "./apidocs-util";
+
+const ApiDocsArea = () => {
+  const client = useLanguageServerClient();
+  const [apidocs, setApiDocs] = useState<ApiDocsResponse | undefined>();
+  useEffect(() => {
+    const load = async () => {
+      if (client) {
+        await client.initialize();
+        const docs = await apiDocs(client);
+        pullModulesToTop(docs);
+        setApiDocs(docs);
+      }
+    };
+    load();
+  }, [client]);
+  return (
+    <Box height="100%" p={2}>
+      {apidocs ? (
+        <ModuleDocs docs={apidocs} />
+      ) : (
+        <Spinner label="Loading API documentation" alignSelf="center" />
+      )}
+    </Box>
+  );
+};
+
+const ModuleDocs = ({ docs }: { docs: ApiDocsResponse }) => {
+  return (
+    <>
+      <Accordion allowToggle wordBreak="break-word">
+        {sortBy(Object.values(docs), (m) => m.fullName).map((module) => (
+          <AccordionItem key={module.id}>
+            <AccordionButton
+              fontSize="xl"
+              _expanded={{ fontWeight: "semibold" }}
+            >
+              <Box flex="1" textAlign="left" mr={3}>
+                {module.fullName}
```

Comment:
This should probably be a heading of some sort. We should review everything beneath it for semantic html use.